source/Playnite.FullscreenApp/FullscreenApplication.cs

```diff
@@ -94,7 +98,6 @@ public override bool Startup()
             SendUsageDataAsync();
 #pragma warning restore CS4014
             ProcessArguments();
-            InitializeAudio();

```

Comment:
Just thought I'd point out that this is specifically being moved because `OpenMainViewAsync` is an async method but it's not being awaited. So the problem I'd end up having is that the intro video would start playing which is awaited inside `OpenMainViewAsync`, but since `OpenMainViewAsync` itself is not awaited, the rest of the app initialization including `InitializeAudio` would occur. This end result of this is that I'd hear audio from the intro video plus the general playnite audio simultaneously, which was not desired.